theme/layouts/_navigation.ejs

```diff
@@ -1,9 +1,23 @@
+<nav id="main-navigation">
+  <% function recNav(navig) { %>
+    <ul>
+      <% for ({title, link, sub_items} of navig) { %>
+        <li <%- sub_items ? 'class="has-children"' : '' %>>
```

Comment:
Detail/consistency: In the current code we are using double quotation marks for the string literals, let's keep it consistent, one way of the other, so you can write `<%- sub_items ? "class='has-children'" : "" %>` instead.

Reply:
BTW: If you feel like using `'` instead of `"` I'm happy to switch to.